builder/builder-next/executor_unix.go

```diff
@@ -88,23 +91,26 @@ func (iface *lnInterface) init(c libnetwork.NetworkController, n libnetwork.Netw
 func (iface *lnInterface) Set(s *specs.Spec) {
 	<-iface.ready
 	if iface.err != nil {
+		logrus.Errorf("failed setting networking spec: %+v", iface.err)
```

Comment:
For consistency with other errors (easier to `grep`):

```suggestion
		logrus.Errorf("failed to set networking spec: %+v", iface.err)
```

or perhaps this (but I guess we'd loose the `%+v` verbosity)

```suggestion
		logrus.WithError(iface.err).Error("failed to set networking spec")
```